web/src/display-defaults/tasks/TaskDefaults.ts

```diff
@@ -8,4 +8,6 @@ export const TaskDefaults: Record<string, string> = {
   unlockedBySingular: "Before starting this task, we recommend that you take a look at:",
   unlockedByPlural: "Before starting this task, we recommend you take a look at:",
   loadingTaskDependencies: "Loading task dependencies…",
+  requiredTagText: "Required",
+  requiredIconAltText: "Exclamation mark",
```

Comment:
Do we want/need alt text for this icon? My understanding is that alt text should convey meaning, not a description of what's on the page. So if the exclamation icon is only there to provide a visual cue, I feel like having a screenreader read out "exclamation mark" would be an unpleasant experience.

Reply:
I guess since there is a tooltip with a label that would get read by screen readers, there's no need for an alt text here then? I imagine a more semantic alt text would end up being the same as the tooltip label.